### PR TITLE
Add in various volunteer extension permissions to the demo role in th…

### PR DESCRIPTION
**Pull request by seamuslee001**
…e demo builds

see #632 

ping @demeritcowboy @homotechsual

### CODE REVIEW

File: app/config/drupal-demo/install.sh [hunk 1]
```diff
@@ -112,7 +112,19 @@ EOPERM
...
+    drush scr "$PRJDIR/src/drush/perm.php" <<EOPERM
+      role 'civicrm_webtest_user'
+      add 'create volunteer projects'
+      add 'edit own volunteer projects'
+      add 'edit all volunteer projects'
+      add 'log own hours'
+      add 'edit volunteer project relationships'
+      add 'edit volunteer registration profiles'
+      add 'delete own volunteer projects'
+      add 'delete all volunteer projects'
```
Comment (demeritcowboy) [inline app/config/drupal-demo/install.sh:127]: Thanks @seamuslee001 - I haven't tested yet but wondering if it should also have 'administer CiviDiscount' ?

Comment (demeritcowboy) [reply app/config/drupal-demo/install.sh:127]: Or maybe not necessary - it looks like CiviDiscount always treats administer CiviCRM as equivalent.

**Comment by demeritcowboy** [comment]
I sort of tested it but I can't completely verify since buildkit isn't fully working for me and I don't really want to spend a lot of time on this. I can see that it appears to have set up the db role_permission entries correctly (drupal 7), and cross-ref'ing with https://github.com/civicrm/org.civicrm.volunteer/blob/46b5a9f39b1fa139d7060f3b1d5a0d76e7d636f6/CRM/Volunteer/Permission.php#L16 they appear to match up.

```
select p.rid,r.name,p.permission from role_permission p inner join role r on r.rid = p.rid where p.permission like '%volu%' or p.permission ='log own hours';
+-----+----------------------+--------------------------------------+
| rid | name                 | permission                           |
+-----+----------------------+--------------------------------------+
|   1 | anonymous user       | register to volunteer                |
|   2 | authenticated user   | register to volunteer                |
|   4 | civicrm_webtest_user | create volunteer projects            |
|   4 | civicrm_webtest_user | delete all volunteer projects        |
|   4 | civicrm_webtest_user | delete own volunteer projects        |
|   4 | civicrm_webtest_user | edit all volunteer projects          |
|   4 | civicrm_webtest_user | edit own volunteer projects          |
|   4 | civicrm_webtest_user | edit volunteer project relationships |
|   4 | civicrm_webtest_user | edit volunteer registration profiles |
|   4 | civicrm_webtest_user | log own hours                        |
|   4 | civicrm_webtest_user | register to volunteer                |
+-----+----------------------+--------------------------------------+
```

**Comment by seamuslee001** [comment]
i'm going to merge based on the above

**Pull request merged**